modules/nextflow/src/main/groovy/nextflow/util/ConfigHelper.groovy

```diff
@@ -263,8 +263,9 @@ class ConfigHelper {
         int i=0
         for( Map.Entry entry : map.entrySet() ) {
             if( i++>0 ) result.append(', ')
+            result.append('\'')
             result.append(entry.key)
-            result.append(":")
+            result.append("':")
             result.append(render0(entry.value))
```

Comment:
Is this really needed? Quotes should only be needed when using keys that are not valid identifiers e.g. `some-value`, `0zero`, 'with blank`

Reply:
This is technically not needed, but it makes the format consistent. I'll use `warp1` instead.

Reply:
Nice